app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, source_path, destination_path = command.split(" ")
+
+    if len(command.split()) == 3 and "mv" in cmd:
+
+        if "/" not in destination_path:  # because name cannot contain "/"
+            os.rename(source_path, destination_path)
+
+        os.makedirs(os.path.split(destination_path)[0])
```

Comment:
As for me this code is not needed. Can you check?
If needed checking for / is so bad. Because in some OS it will not work. Read about os.path.split

Reply:
> As for me this code is not needed. Can you check? If needed checking for / is so bad. Because in some OS it will not work. Read about os.path.split

for `move_file("mv rename.txt rename2.txt")` result will be: 
['mv', 'rename.txt', 'rename2.txt']
('', 'mv rename.txt rename2.txt')

and for `move_file("mv move.txt first_dir/second_dir/move2.txt")`:
['mv', 'move.txt', 'first_dir/second_dir/move2.txt']
('mv move.txt first_dir/second_dir', 'move2.txt')

i can replace "/" check by `if os.path.split(command)[0] == ""`